src/expressions/block-expr.md

```diff
@@ -117,6 +117,37 @@ loop {
 }
 ```
 
+## `const` blocks
+
+> **<sup>Syntax</sup>**\
+> _ConstBlockExpression_ :\
+> &nbsp;&nbsp; `const` _BlockExpression_
+
+A *const block* is a variant of a block expression which evaluates in the compile time instead of in the run time.
```

Comment:
Is it a requirement for other compilers to actually evaulate the expression at compile time?

I would think the primary definition would be that it is a block that's a const context.

Reply:
The const eval doc does mention that exprs in const context are evaluated at compile time, so I don't see an issue here.